### DISCUSSION TITLE
"DWMBlurGlass 3.0" from Discord has a token logger

### DISCUSSION THREAD

**Discussion by catneptune** [Maplespe/DWMBlurGlass#319] [General]
# THIS DOES NOT EFFECT PEOPLE WHO GET DWMBLURGLASS FROM THIS REPOSITORY

I wanted to tell y'all that Alcatel (https://github.com/arukateru/) has made a fake DWMBlurGlass 3.0 beta and a fake Discord, also going as far as impersonating ALTaleX and Maplespe and uploaded a cookie logger that steals all Chromium-based browsers data, this includes Chrome, Edge, Brave, Opera (GX), Yandex. This does not include Firefox, this also steals Steam, Riot Games and Telegram data, takes system information (HWID, Username, Disk, MAC, IP), files, and adds itself to startup.
A screenshot of the message:

![image](https://github.com/Maplespe/DWMBlurGlass/assets/144282272/96f4bf55-d3a0-4af5-8900-af92df133e73)

Be careful out there.

**Comment by ericoepico69** [comment]
this shit is getting worse istg

**Comment by leetftw** [comment]
This community is pathetic sometimes. I really can't believe we have to deal with bullshit like this sometimes. Bit of advice for everyone: do not trust stuff because it comes from a known community member, always make sure to check whatever you downloaded. Stay safe everyone!

**Comment by kfh83** [comment]
I can't say this enough but for anyone who was in the "DWMBlurGlass" discord server, PLEASE RESET ALL OF YOUR PASSWORDS, LOG OFF ANY OTHER DEVICES AND REINSTALL WINDOWS.

I can confirm he snatched my discord token (atleast) and i'm not sure about any other data like firefox

**Reply by funn-hue** [reply]
Nah i think being in the server didn't cause any harm, only if someone downloaded the file, then they should do it

**Reply by kfh83** [reply]
I think if you were in the server there's a good chance you downloaded it, you know what i mean

**Comment by calex7z** [comment]
He used a token logger called [CStealer](https://github.com/can-kat/cstealer)

**Reply by kfh83** [reply]
^ Supposedly it can't snatch firefox information but still for anyone who was affected, just to be safe, please reset ALL of your passwords and log out of all your sessions. Also reinstall windows.

**Comment by 7kt4** [comment]


**Reply by catneptune** [reply]
shut the fuck up, we all know it was you alcatel. it was your main account and we even have screenshots of you testing out the token logger.

**Reply by catneptune** [reply]
![lol](https://github.com/Maplespe/DWMBlurGlass/assets/144282272/bb58252d-f0d9-4073-a504-99360055ae37)

**Reply by kawapure** [reply]
We were all there in the server. You fumbled over and over again. We all can confirm it was you.

**Reply by catneptune** [reply]
dude, alcatel who the fuck says that shit? it's literally you, it had the same user ID as you, don't try gaslighting because you're not fooling anyone

**Reply by ImSwordQueen** [reply]
![image](https://github.com/Maplespe/DWMBlurGlass/assets/52015098/cd6434f9-d81e-4a16-94fa-ee4a67e0f5bc)
Sure buddy.

**Reply by kawapure** [reply]
@arukateru It was literally your account with verifiable evidence. You were not impersonated. This is verifiably you. You impersonated the creator of DWMBlurGlass and spread malware.

**Reply by leetftw** [reply]
Trying to cover his ass ahhhh alcatel

**Reply by kfh83** [reply]
The fact you really did this shit and tryna cover it up is insane its like doing a school shooting running out guns out hiding in a bush and saying you aint do it

**Reply by catneptune** [reply]
i don't think vaporvance, callyham, theinsane101 would say the same dude. you token logged them. you renamed them to "Big Alcatel fan", you called dulappy the hard r. what the fuck is wrong with you?????
![image](https://github.com/Maplespe/DWMBlurGlass/assets/144282272/7ebcce86-9ed5-4221-b354-d91f8f0ec344)

**Reply by catneptune** [reply]
oh now you're deleting your shit alcatel? pussy

**Reply by UnknownPolish** [reply]
Dude stop lying,it was YOU who did it. No impersonators. Lying is only gonna make the situation worse, you know.

**Comment by Randomuser8219** [comment]
First he was a msstyles thief, now THIS? I'm using Aero10 theme by Vaporvance from now on.

**Comment by inteneich** [comment]
at first, this guy just blatantly stole a theme and now he did this. what a idiot I was when I sided with him once yesterday. really sometimes I forget to think clear

**Comment by Sand216** [comment]
I can't believe this... What a horrible thing for someone who is active in this community to do. Hopefully not active here much longer!

**Comment by kwan-vini** [comment]
to: alcatel
from: me

![image](https://github.com/Maplespe/DWMBlurGlass/assets/118130060/15f915c6-da27-41c5-bd93-54405bd97382)

**Comment by kkirta** [comment]
![lol](https://github.com/Maplespe/DWMBlurGlass/assets/37386673/da1b03a5-648e-49c8-a4a4-7309bd2d979a)

**Reply by Sand216** [reply]
What sort of harassment has Alcatel faced before this? Am I missing something? As far as I know this community is very supportive and respectful as long as you are respectful back.

**Reply by Undisputed00x** [reply]
@ven-ox Alcatel how many accounts do you have in your disposal ?, No matter what you say to save your ass you are a disgrace you did damage people's privacy if not their lifes

**Reply by iminonet** [reply]
not only did alcatel compromise venoxy's discord but the github account too ._.
wow alcatel good job you are a fucking faliure

**Reply by kfh83** [reply]
He's just relaying the message, this was posted somewhere

**Reply by leetftw** [reply]
ClassicServ copypasta real

**Reply by kfh83** [reply]
![caption](https://github.com/Maplespe/DWMBlurGlass/assets/85626296/c3433f4c-f463-4432-9987-3e66aad6683e)

**Comment by Clasurol** [comment]
What they did was horrendous, but  not going to lie, I'm finding the messages telling this person to kill themselves extremely tasteless. You all are better than that.

**Reply by Sand216** [reply]
Exactly! I was thinking about saying something, but didn’t want to get ridiculed.

Alcatel did something bad, but that doesn’t mean we should fight fire with fire.

**Comment by leetftw** [comment]
I quickly want to address something. I understand and agree with the fact that what Alcatel did is unacceptable. It is disgraceful behavior and there is no excuse for it. But I see that a lot of people are throwing massive hate towards him for it in ways that are extremely hurtful. If you want to call someone out for what they did, fine. But telling people to kill themselves and that they won’t be missed if they were somehow gone is terrible. I am not sure that I entirely believe Alcatel when he is saying he is not doing well mentally because of the community, but if it is true you are only making things worse. It is worse than anything Alcatel could have possibly done within the community. And again, for clarity: I myself think Alcatel is an asshole, so there is no bias here.

Quick addition: if you are having mental problems make sure you reach out to the proper channels. If you are in a seriously bad mental state call a suicide hotline. In the US, you can call 988. For the Netherlands this is 113. In other countries, look the number up on Google.

(FYI I’m posting this in multiple places)

**Comment by UnknownPolish** [comment]
This is ridiculous on another level,why do you want to scam People in Modding Community and then pretend they are a fan of you??? This is not only too far,but also uncalled for and Crossing the Line. You're racist,you are Transphobic,noone would be a fan of you if you are racist and Transphobic. Look I joined this Modding Community since February 2024 and I have not realized there are Scammers here, I never used DWMBlurGlass and probably never will (I probably will in the Future), and I was never excited for "DWMBlurGlass 3.0" and never even joined their ""DWMBlurGlass Discord Server"", but I did not realize there are asswipe scammers here. If you fall for a scam you are an idiot I know,but I still feel bad for the people involved in that situation. Besides Impersonation of People is also bad, Alcatel,if you are read this,stop being an Asswipe and hacking people who did nothing to you. Become a normal Person in this Community.

That is my Opinion on all of this.

**Comment by Maplespe** [comment]
To avoid this kind of matter from happening again, please do not download the software from unofficial addresses!

!!!**We also don't have any official Discord**!!!

We only distribute software on [Github](https://github.com/Maplespe/DWMBlurGlass/releases), [Bilibili](https://space.bilibili.com/87195798) and [winmoes](https://winmoes.com/).

As well, any new versions for testing are pushed to the test branch first, rather than releasing binaries in advance.

I've added the caveats mentioned above to ReadMe in case something similar happens.
